src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -190,6 +190,7 @@ static bool updateForward
 
   ciP->outMimeType  = outMimeType;
   cleanPayload      = (char*) payload.c_str();
+  std::string         url;
```

Comment:
Same comment here: https://github.com/telefonicaid/fiware-orion/pull/3749/files#r636880073

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3749/commits/d4e75f554d1327a17ec1235529378567fd5c4454